### DISCUSSION TITLE
Installation path issues

### DISCUSSION THREAD

**Discussion by mieyeaa** [anyproto/contributors#67] [Any-Block & Anytype-Heart]
Multiple path installation methods, when will it appear

**Comment by fuksman** [comment]
Hey @mieyeaa, sorry, I don't understand the issue. Could you please provide more details about the issues you're facing and how they are connected to Any-Block or Anytype-Heart?

**Reply by mieyeaa** [reply]
Such as "ZIP", which is a ready-to-use "installation method".

**Reply by michael1maass** [reply]
maybe I can support this with a usecase: 

in a windows production /company environment, policies are in force that prohibit program execution from locations other than c:\program files and the respective subdirectories. Therefore any installation must be global, with administrative rights. The actual windows installer of anytype does not offer that choice, but installs into the current user environment (c:\users\username\apdata\local\programms\anytype.....) 

On windows you should either offer an option to install global or do it right away. 

With that comes another restriction: as a normal user, you can not write into c:\program files\.... for obvious security reasons. So user related data has still to go into the AppData path, or a hidden (.xxxx) Folder on the c:\users\username\level Folder. 

as of now, I have to do the install, and move the program folder into c:\program files\ which is a pain..., since there are ways to take care of that. 

I am lurking over from Logseq btw, which has the same issue, plus some syncing policies that make it useless in a corporate environment. 

I hope that was understandable. I'm glad to answer further questions an check out your approaches to this challange in our env. 

regards

Michael
